govcd/lbapprule_test.go

```diff
@@ -57,6 +57,17 @@ func (vcd *TestVCD) Test_LBAppRule(check *C) {
 
 	check.Assert(createdLbAppRule.Script, Equals, lbAppRuleConfig.Script)
 
+	// Test that we can extract a list of LB app rules, and that one of them is the rule we have got when searching by name
+	lbAppRules, err := edge.GetLbAppRules()
+	check.Assert(err, IsNil)
```

Comment:
check that `lbAppRules` is not nill is missing

Reply:
done